projects/js-packages/components/components/dialog/stories/index.tsx

```diff
@@ -56,6 +56,12 @@ InterstitialJetpackBoost.args = {
 			pricing={ {
 				isFree: true,
 			} }
+			className={ '' }
+			icon={ '' }
+			isCard={ false }
+			supportedProducts={ [] }
+			hasRequiredPlan={ false }
+			addProductUrl={ '' }
```

Comment:
These shouldn't be required. If that's the case, we can make those props optional in JS docs of `ProductOffer` component. For example, we can replace:
`@param {string} props.className`
with
`@param {string} [props.className]`

Reply:
yes, I agree. I didn't want to address it here, though. I suggest doing it in a follow-up one.